MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/CircleLayerClusteringActivity.java

```diff
@@ -137,7 +144,7 @@ private void addClusteredGeoJsonSource() {
 
     // Add a new source from the GeoJSON data and set the 'cluster' option to true.
     try {
-      mapboxMap.addSource(
+      mapboxMap.getStyle().addSource(
```

Comment:
maybe change addClusterGeoJsonSource signature to take Style as parameter? This makes it more clear to readers that a style is required to call this method.

Reply:
fixed